tests/unit/lms/services/vitalsource_test.py

```diff
@@ -78,7 +78,7 @@ def lti_user(self):
 
```

Comment:
Don't you also need to move the test file to `tests/unit/lms/services/vitalsource/client_test.py`? And you'll need to add `tests/unit/lms/services/vitalsource/__init__.py` too

Reply:
You are right, done :+1: